src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,87 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class WorkWithFile {
+    private List<String> listOfFilesData = new ArrayList<>();
 
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder dataToWrite = new StringBuilder();
+        readDataFromFile(fromFileName);
+        String[] finalArrayOfData = generateArrayWithCorrectData(listOfFilesData, new String[2]);
+        int[] finalSumOfEachElementOfArrayData =
+                generateSumForEachElementOfCorrectDataArray(
+                        listOfFilesData,
+                        finalArrayOfData,
+                        0,
+                        0);
+        dataToWrite.append(finalArrayOfData[1]).append(",")
+                .append(finalSumOfEachElementOfArrayData[1])
+                .append(System.lineSeparator())
+                .append(finalArrayOfData[0]).append(",")
+                .append(finalSumOfEachElementOfArrayData[0])
+                .append(System.lineSeparator())
+                .append("result").append(",")
+                .append(finalSumOfEachElementOfArrayData[1] - finalSumOfEachElementOfArrayData[0]);
+        writeDataToFile(toFileName, dataToWrite);
+    }
+
+    public void writeDataToFile(String fileName, StringBuilder dataToWrite) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(fileName))) {
+            writer.write(dataToWrite.toString());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't find a file: " + fileName, e);
+        }
+    }
+
+    public void readDataFromFile(String fileName) {
+        String line;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            while ((line = reader.readLine()) != null) {
+                listOfFilesData.addAll(Arrays.asList(line.split(",")));
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from file: " + fileName, e);
+        }
+    }
+
+    public String[] generateArrayWithCorrectData(List<String> data, String[] result) {
```

Comment:
```suggestion
    public String createReport(List<String> data) {
```
The method should look like this. It should take data from a file, parse it, and create a string report, that later is going to be just put in the output file